src/iuse_actor.cpp

```diff
@@ -1704,3 +1704,97 @@ bool musical_instrument_actor::can_use( const player *p, const item*, bool, cons
 
     return true;
 }
+
+iuse_actor *holster_actor::clone() const
+{
+    return new holster_actor(*this);
+}
+
+void holster_actor::load( JsonObject &obj )
+{
+    holster_prompt = obj.get_string("holster_prompt", _("Holster item"));
```

Comment:
The string from JSON won't get translated. It is not translated here, nor below in the `add_msg` calls. Other `iuse_actor` handle that by loading and storing the untranslated string (with an empty string as default) and translating when printed, or using a fixed string if the stored message is empty.

That way is not really good, but it works. Nicer would be to have a `get_translated_string` in the JSON classes, that automatically translated *non-empty* strings and allows for a default value. That gets complicated.